src/components/Flashcard/Flashcard.js

```diff
@@ -1,100 +1,132 @@
-    }
+    };
 
     const movingInsidePreviousQuestions = (direction) => {
         let newCount = countQuestions + direction;
-        let newCurrentQuestion = questions.find(question => question.id === previousQuestions[newCount - 1]);
+        let newCurrentQuestion = questions.find(
+            (question) => question.id === previousQuestions[newCount - 1]
+        );
         setCurrentQuestion(newCurrentQuestion);
         setCountQuestions(newCount);
-    }
+    };
 
     const drawNextQuestion = () => {
         let nextId;
-        while(nextId === undefined || previousQuestions.includes(nextId)){
-            let randomCategory = Math.floor(Math.random() * 100); 
+        while (nextId === undefined || previousQuestions.includes(nextId)) {
+            let randomCategory = Math.floor(Math.random() * 100);
             if (randomCategory < 45) {
                 nextId = randomIdFromCategory(unknown);
-            }else if(randomCategory < 85){
+            } else if (randomCategory < 85) {
                 nextId = randomIdFromCategory(all);
-            }else{
+            } else {
                 nextId = randomIdFromCategory(known);
             }
         }
-        let nextQuestion = questions.find(question => question.id === nextId);
+        let nextQuestion = questions.find((question) => question.id === nextId);
         setCurrentQuestion(nextQuestion);
 
         let newPrevious = previousQuestions;
         newPrevious.push(nextId);
         setPreviousQuestions(newPrevious);
 
         setCountQuestions(countQuestions + 1);
-    }
+    };
 
     const randomIdFromCategory = (category) => {
-        if(category.length > 0){
+        if (category.length > 0) {
             let numberToDraw = category.length;
             let drawId = Math.floor(Math.random() * numberToDraw);
             return category[drawId];
         }
-    }
+    };
 
     return (
         <div className={styles.flashcardView}>
             <div className={styles.flashcardPlusButtons}>
-                { countQuestions > 1 
-                    ?   <Button onClick={() => backToPrevious()} className={ 'iconButton' }><i><FaChevronCircleLeft/></i></Button> 
-                    :   <p> </p> 
-                }
-                <CardContent question={ currentQuestion } addToUserBase={ addToUserBase } known={known} unknown={unknown}/>
-                { countQuestions < 12 
-                    ?   <Button onClick={() => nextQuestion()} className={'iconButton'}><i><FaChevronCircleRight/></i></Button> 
-                    :   <Button onClick={() => setShowIntro(true) } className={'iconButton'}><i><FaUndo/></i></Button> 
-                }
+                {countQuestions > 1 ? (
+                    <Button onClick={() => backToPrevious()} className={'iconButton'}>
+                        <ArrowLeftIcon className={styles.flashcardIcon} />
+                    </Button>
+                ) : (
+                    <p> </p>
```

Comment:
Can't be null instead?

Reply:
nope, because empty p tag is creating come container with small width which will prevent content from moving to the side in the first flashcards, in the second flashcard there will be a arrow to come back to first one.